extension/src/utils/AspireTerminalProvider.ts

```diff
@@ -135,6 +136,15 @@ export class AspireTerminalProvider implements vscode.Disposable {
             env.ASPIRE_EXTENSION_DEBUG_RUN_MODE = noDebug === false ? "Debug" : "NoDebug";
             env.DEBUG_SESSION_INFO = JSON.stringify(getRunSessionInfo());
             env.ASPIRE_EXTENSION_CAPABILITIES = getSupportedCapabilities().join(',');
+
+            // if DCP debug logging is enabled, set DCP-specific logging environment variables
+            const dcpDebugLoggingEnabled = vscode.workspace.getConfiguration('aspire').get<boolean>('enableAspireDcpDebugLogging', false);
```

Comment:
DCP logs for a session will be put in ${workspaceRoot}/.aspire/dcp/logs-{session}. DCP's default logging is not sufficient to actually find issues most of the tim